static-build/index.tsx

```diff
@@ -28,15 +28,38 @@ const toOutput: Output = {
   'faqs/index.html': renderPage(<FAQPage />),
 };
 
+function getPrevOrNextData(testEntry: [string, Test]): PaginationData {
+  return {
+    link: './../' + testEntry[0] + '/',
+    meta: {
+      title: testEntry[1].meta.title,
+      shortDesc: testEntry[1].meta.shortDesc,
+    },
+  };
+}
+
 function addTestPages(tests: Tests, basePath = '') {
-  for (const [testPath, test] of Object.entries(tests)) {
+  const testEntries: [string, Test][] = Object.entries(tests);
+  const len = testEntries.length;
+  testEntries.forEach(([testPath, test], i) => {
```

Comment:
the for loop can provide an index as well no? we tried to keep the code style in for of loops, curious why you swapped it here?

Reply:
@argyleink I am really sorry, I should have followed your style.
I also really appreciate the time you give to my code, dude, Thanks.